iwyu_output.cc

```diff
@@ -1529,6 +1529,17 @@ void IwyuFileInfo::CalculateIwyuViolations(vector<OneUse>* uses) {
   VERRS(6) << "--- Calculating IWYU violations for "
            << GetFilePath(file_) << " ---\n";
 
+  // Remove any uses part of an override decl -- they are covered for by the
+  // base class from where they were overridden, and the header for the base
+  // class must already be included because we derive from it.
+  for (OneUse& use : *uses) {
+    if (use.is_part_of_override_decl()) {
+      VERRS(5) << "Ignoring use of " << use.symbol_name() << " ("
+               << use.PrintableUseLoc() << "): re-exported from override\n";
```

Comment:
> re-exported from override

I find the term 'override' confusing. Depending on the context it might refer to the methods being overriden, or to the one that overrides them.

Why not 're-exported from base'?

Reply:
I went with `re-exported from base class`